cranelift/codegen/src/isa/s390x/inst/mod.rs

```diff
@@ -3621,6 +3624,7 @@ impl MachInstLabelUse for LabelUse {
             // This can address any valid CodeOffset.
             LabelUse::BranchRIL => 0x7fff_ffff,
             LabelUse::PCRel32 => 0x7fff_ffff,
+            LabelUse::PCRel32Dbl => 0x7fff_ffff,
```

Comment:
Given the shift, is the range actually one bit larger than what's specified here?

Reply:
The range is actually +/- 4GB, not +/-2 GB.   But that apparently cannot be represented as CodeOffset anyway, so I simply accept all valid CodeOffsets here.

Reply:
Could this return `0xffff_ffff` as a placeholder? One day I think it'd be good to represent some code offsets in this area as a 64-bit integer, but that I think is a much bigger change.

Reply:
I tried that when adding BranchRIL (which has the same range) initially, and it caused failures (some kind of overflows in common code), so that's what I ended up with.   I guess this could be revisited at some point, but then we should do it for all affected LabelUse types, not just PCRel32Dbl.

Reply:
Can the overflow be fixed with a `saturating_add` or something like that somewhere? If not can the comment be updated that this should be 4GB but it's specified as 2GB for now?

Reply:
That would probably help, I'll give it a try.   It's still better to do that in a separate PR I think.